README.md

```diff
@@ -1,5 +1,10 @@
 # rSFSTEP2
 
+# Cloning the repository:
+```
+git clone --branch master https://github.com/DrylandEcology/rSFSTEP2.git
+```
+
 # Instructions for running rSFSTEP2
```

Comment:
These instructions don't work for me. I follow instructions for a local machine, clone, change directory, and run the script `cloneSTEPWAT2.sh` for step 5 (using the example siteids 1 and 2). When I execute step 6 with `./generate_rSFSTEP2_structure.sh . 2 1`, the script copies the entire folder iteratively until I have to force quit the process. I am confused because it seems that I should run the script from `.`, but this is also the location/path of `R_program/` -- apparently I do something wrong. Maybe providing the code that runs the example sites would make it clearer?

Reply:
@dschlaep I see how this was confusing. I will update accordingly, but for two sites and 1 climate scenario you need to execute:

`./generate_rSFSTEP2_structure.sh R_program 2 1 `

Reply:
addressed by https://github.com/DrylandEcology/rSFSTEP2/commit/aa9d435583325d195f1dceff2b1cb0109056b5c3

Reply:
Thanks!